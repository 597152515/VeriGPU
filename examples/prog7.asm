li x2, 0x40
li x1, 123
outr x1
outr x2
sw x1, 0(x2)
outloc 0x40

li x2, 0x54
li x1, 111
sw x1, 0(x2)
outloc 0x54

outloc 0x40
outloc 0x54

halt
