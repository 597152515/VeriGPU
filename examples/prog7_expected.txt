out 0 0000007b 123
out 1 000000f0 240
out 2 0000007b 123
out 3 0000006f 111
out 4 0000007b 123
out 5 0000006f 111